src/extension.ts

```diff
@@ -535,6 +541,40 @@ async function explain(obj: any, field: string): Promise<string | null> {
     });
 }
 
+async function browsePipeline(node: CloudExplorerTreeNode): Promise<void> {
+    if (await isAzurePipelinesExtensionInstalled()) {
+        await executeAzurePipelineExtensionCommand(browsePipelineCommand, node);
+    }
+}
+
+async function configurePipeline(node: CloudExplorerTreeNode): Promise<void> {
+    if (await isAzurePipelinesExtensionInstalled()) {
+        await executeAzurePipelineExtensionCommand(configurePipelineCommand, node);
+    }
+}
+
+async function isAzurePipelinesExtensionInstalled(): Promise<boolean> {
+    const pipelinesExtension = vscode.extensions.getExtension(azurePipelinesExtensionId);
+    if (!pipelinesExtension) {
+        try {
+            vscode.window.showWarningMessage('Please install/enable `Deploy to Azure` extension and then retry.');
+        } catch (err) {
+            return false;
+        }
+        await vscode.commands.executeCommand('extension.open', azurePipelinesExtensionId);
```

Comment:
what does this do? will it open extension's marketplace page?

Reply:
It opens the following page
![image](https://user-images.githubusercontent.com/58769601/93568957-86d5b680-f9ae-11ea-83d3-4f78eea6993c.png)